src/test/resources/features/booking_details.feature

```diff
@@ -83,21 +83,11 @@ Feature: Booking Details
     When a basic offender booking request is made with booking id "-13"
     Then resource not found response is received from bookings API
 
-  Scenario: Request for specific offender as system user can return data even though booking is inactive
```

Comment:
Rewrote these two to check inactive bookings (the rest will need rewriting at a later time)